pipeline/test_beam_tables.py

```diff
@@ -117,15 +117,22 @@ def test_get_full_table_name(self) -> None:
 
   def test_read_scan_text(self) -> None:  # pylint: disable=no-self-use
     """Test reading lines from compressed and uncompressed files"""
-    p = TestPipeline()
-    pipeline = beam_tables._read_scan_text(
-        p, ['pipeline/test_results_1.json', 'pipeline/test_results_2.json.gz'])
-
-    beam_test_util.assert_that(
-        pipeline,
-        beam_test_util.equal_to([
-            'test line 1.1', 'test line 1.2', 'test line 2.1', 'test line 2.2'
-        ]))
+    with TestPipeline() as p:
```

Comment:
You can delete this test if you delete `_read_scan_text`.

Reply:
I think this is a valuable thing to be testing. (Both that we can read files correctly and that we can read both zipped and unzipped files transparently.) We're not testing this functionality anywhere else in the unit tests, only the e2e tests which are substantially slower.